### PR TITLE
Fixes Create Command Report admin verb & it can now have a custom origin, removes command_name and cleans related code.

### PR DESCRIPTION
**Pull request by Yellow-Mushroom**
## About the PR 

This PR allows for the Create Command Report verb that allowed admins to make centcomm announcements and what not to first off be fixed since I believe I may had broken it. But second off it makes it significantly more robust, by allowing admins to input where the message comes from, instead of making it strictly come from Central Command. (also adds admin changelog explaining the change)

It also removes command_name and some references in unused/very old areas of the code. I'm not sure why I didn't remove it in its entirety last time as it only served to cause issues and create confusion.

**NEW CHANGE SINCE LAST REVIEW: Fixes issue where the bridge button alert came from the frontier instead of the station!!**

## Why's this needed? 

Create Command Report is used for a lot of good RP events, this will allow for more variety in ease in this. It also fixes the verb to no longer use command_name.

Command_Name has been removed for the purpose of cleanliness, since it was causing issues and had no purpose to remain in the code except to create confusion.

### CODE REVIEW

File: code/procs/command_alert.dm [hunk 1]
```diff
@@ -67,7 +67,7 @@
 				text = replacetext(text, "%mrand_name%", rand_mob_mult.name)
 				text = replacetext(text, "%mrand_job%", rand_mob_mult.job ? rand_mob_mult.job : "space hobo")
 
-				boutput(M, "<h1 class='alert'>[command_name()] Update</h1>")
+				boutput(M, "<h1 class='alert'>[ALERT_GENERAL]</h1>")
```
Comment (ZeWaka) [inline code/procs/command_alert.dm:70]: did you mean to remove Update

Comment (Yellow-Mushroom) [reply code/procs/command_alert.dm:70]: I didn’t, the new alerts contains update/respective titles

Comment (ZeWaka) [reply code/procs/command_alert.dm:70]: then why didn't you remove it everywhere else

Comment (Yellow-Mushroom) [reply code/procs/command_alert.dm:70]: Oh?? Hold on I’ll need to peep? It’s late rn but I thought it was removed on all the others but I may be misremembering.

Comment (Yellow-Mushroom) [reply code/procs/command_alert.dm:70]: Oh yep I entirely missed that for this PR ahahaha. Good catch, I’ll fix it tomorrow

**Comment by Yellow-Mushroom** [comment]
Fixed the double update issue + fixes an issue of the bridge button having the alert come from the frontier authority instead of the station

File: code/WorkInProgress/construction/event_controller.dm [hunk 2]
```diff
@@ -122,7 +122,7 @@
 
 	proc/early_warning()
 		for_by_tcl(C, /obj/machinery/communications_dish)
-			C.add_centcom_report("[command_name()] Update", early_warning_text)
+			C.add_centcom_report("[ALERT_GENERAL]", early_warning_text)
```
Comment (pali6) [inline code/WorkInProgress/construction/event_controller.dm:125]:
```suggestion
			C.add_centcom_report(ALERT_GENERAL, early_warning_text)
```

File: code/WorkInProgress/construction/event_controller.dm [hunk 3]
```diff
@@ -131,7 +131,7 @@
 
 	proc/set_up()
 		for_by_tcl(C, /obj/machinery/communications_dish)
-			C.add_centcom_report("[command_name()] Update", warning_text)
+			C.add_centcom_report("[ALERT_GENERAL]", warning_text)
```
Comment (pali6) [inline code/WorkInProgress/construction/event_controller.dm:134]:
```suggestion
			C.add_centcom_report(ALERT_GENERAL, warning_text)
```

File: code/datums/gauntlet/crittergauntlet.dm [hunk 4]
```diff
@@ -282,7 +282,7 @@
 		if (current_level > 50)
 			var/command_report = "A Critter Gauntlet match has concluded at level [current_level]. Congratulations to: [moblist_names]."
 			for_by_tcl(C, /obj/machinery/communications_dish)
-				C.add_centcom_report("[command_name()] Update", command_report)
+				C.add_centcom_report("[ALERT_GENERAL]", command_report)
```
Comment (pali6) [inline code/datums/gauntlet/crittergauntlet.dm:285]:
```suggestion
				C.add_centcom_report(ALERT_GENERAL, command_report)
```

File: code/datums/gauntlet/podcolosseum.dm [hunk 5]
```diff
@@ -327,7 +327,7 @@
 		if (score > 10000)
 			var/command_report = "A Pod Colosseum match has concluded with score [score]. Congratulations to: [moblist_names]."
 			for_by_tcl(C, /obj/machinery/communications_dish)
-				C.add_centcom_report("[command_name()] Update", command_report)
+				C.add_centcom_report("[ALERT_GENERAL]", command_report)
```
Comment (pali6) [inline code/datums/gauntlet/podcolosseum.dm:330]:
```suggestion
				C.add_centcom_report(ALERT_GENERAL, command_report)
```

File: code/modules/admin/admin.dm [hunk 6]
```diff
@@ -3189,7 +3189,7 @@ var/global/noir = 0
 
 							if (tgui_alert(src, "Headline: [input2 ? "\"[input2]\"" : "None"] | Body: \"[input]\"", "Confirmation", list("Send Report", "Cancel")) == "Send Report")
 								for_by_tcl(C, /obj/machinery/communications_dish)
-									C.add_centcom_report("[command_name()] Update", input)
+									C.add_centcom_report("[ALERT_GENERAL]", input)
```
Comment (pali6) [inline code/modules/admin/admin.dm:3192]:
```suggestion
									C.add_centcom_report(ALERT_GENERAL, input)
```

File: code/modules/admin/spacebee_extension/commands.dm [hunk 7]
```diff
@@ -296,7 +296,7 @@
 	argument_types = list(/datum/command_argument/string="headline", /datum/command_argument/the_rest="body")
 	execute(user, headline, body)
 		for_by_tcl(C, /obj/machinery/communications_dish)
-			C.add_centcom_report("[command_name()] Update", body)
+			C.add_centcom_report("[ALERT_GENERAL]", body)
```
Comment (pali6) [inline code/modules/admin/spacebee_extension/commands.dm:299]:
```suggestion
			C.add_centcom_report(ALERT_GENERAL, body)
```

**Review by pali6 — APPROVED** [review]

**Pull request merged**